components/CollectivesWithData.js

```diff
@@ -4,12 +4,42 @@ import { gql } from '@apollo/client';
 import { graphql } from '@apollo/client/react/hoc';
 import { Button } from 'react-bootstrap';
 import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
 
 import CollectiveCard from './CollectiveCard';
 import Error from './Error';
 
 const COLLECTIVE_CARDS_PER_PAGE = 10;
 
+const CollectivesContainer = styled.div`
+  :global(.loadMoreBtn) {
```

Comment:
I don't think `:global` works with styled components. Does it?

Reply:
Let me double check when I open new PRs. But seemed like it was working. 👍🏼